third_party/py/python_configure.bzl

```diff
@@ -345,7 +345,7 @@ def _python_autoconf_impl(repository_ctx):
         repository_ctx,
         "_python3",
         _PYTHON3_BIN_PATH,
-        "python3",
+        "python3" if not _is_windows(repository_ctx) else "python.exe",
```

Comment:
Would `python3.exe` work here? Or is this naming convention not generally available on windows?

Reply:
I just checked and `python3.exe` works. I'll update the PR.

Reply:
Turns out `python3` didn't work, see PR: https://github.com/ray-project/ray/pull/23246